### PR TITLE
beta version

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
- [DEMO LINK](https://Anton-Zhytonbaiev.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Anton-Zhytonbaiev.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,59 @@
...
+      </div>
+
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star star"></div>
+          <div class="stars__star star"></div>
+          <div class="stars__star star"></div>
+          <div class="stars__star star"></div>
+          <div class="stars__star star"></div>
+        </div>
```
Comment (vladyslav78292) [inline src/index.html:33]: checklist point 3

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,59 @@
...
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image image"
+        src="images/imac.jpeg"
+        alt="macImage"
```
Comment (vladyslav78292) [inline src/index.html:16]: checklist point 13

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,59 @@
...
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
```
Comment (vladyslav78292) [inline src/index.html:9]: would be better if you move second link to another row

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,59 @@
...
+        <div class="number">
+          $2,199
+        </div>
+      </div>
+
+      <a
+          href="#buy"
+          class="card__buy buy"
+          data-qa="hover"
+        >
```
Comment (vladyslav78292) [inline src/index.html:54]:
```suggestion
      <a
        href="#buy"
        class="card__buy buy"
        data-qa="hover"
      >
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:not(:last-child),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
```
Comment (vladyslav78292) [inline src/style.css:75]: in this project should be 2 BEM blocks in your code - card and stars. create new folder inside src, for example "styles". in styles folder create files "card.css" and "stars.css". paste all code , which related with stars in stars.css and import it in this file. same with card block

**Review by vladyslav78292 — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,61 @@
...
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
```
Comment (Denchepornyuk) [inline src/index.html:14]: it's better to use article tag for the card

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,61 @@
...
+      <img
+        class="card__image image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+      >
+      <div class="card__name name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
```
Comment (Denchepornyuk) [inline src/index.html:22]: it should be title, use tag for a title (h1, h2, h3, ...)

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,61 @@
...
+
+      <div class="card__code code">
+        Product code: 195434
+      </div>
```
Comment (Denchepornyuk) [inline src/index.html:26]: don't use div for each element on the page, use more semantic tags

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,61 @@
...
+
+      <div class="card__price price">
+        <div class="letter">
+          Price:
+        </div>
+
+        <div class="number">
+          $2,199
+        </div>
+      </div>
```
Comment (Denchepornyuk) [inline src/index.html:50]: the same, use semantic tags

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,61 @@
...
+
+      <a
+        href="#buy"
+        class="card__buy buy"
+        data-qa="hover"
+      >
+          Buy
```
Comment (Denchepornyuk) [inline src/index.html:57]:
```suggestion
        Buy
```

File: src/index.html [hunk 11]
```diff
@@ -1,13 +1,61 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+      >
+      <div class="card__name name">
```
Comment (Denchepornyuk) [inline src/index.html:20]: do you need `image` and `name` BEM blocks here?

File: src/index.html [hunk 12]
```diff
@@ -1,13 +1,61 @@
...
+          <div class="stars__star star"></div>
+          <div class="stars__star star"></div>
+        </div>
+
+        <div class="card__rate">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price price">
```
Comment (Denchepornyuk) [inline src/index.html:42]: do you need block `price` here?

File: src/index.html [hunk 13]
```diff
@@ -1,13 +1,61 @@
...
+        <div class="letter">
+          Price:
+        </div>
+
+        <div class="number">
```
Comment (Denchepornyuk) [inline src/index.html:47]: why just `number` it should be element from card

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,139 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```
Comment (Denchepornyuk) [inline src/style.css:3]: add `font-family` to html tag

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 14 above)
Comment (Denchepornyuk) [inline src/style.css:3]: you can remove `font-family` from body, you have it in html tag

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**